forge-gui/res/cardsfolder/upcoming/black_market_tycoon.txt

```diff
@@ -0,0 +1,10 @@
+Name:Black Market Tycoon
+ManaCost:R G
+Types:Creature Cat Rogue
+PT:2/2
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigDamage | TriggerDescription$ At the beginning of your upkeep, CARDNAME deals 2 damage to you for each Treasure you control.
+SVar:TrigDamage:DB$ DealDamage | Defined$ You | NumDmg$ X
+SVar:X:Count$Valid Treasure.YouCtrl/Times.2
+A:AB$ Token | Cost$ T | TokenAmount$ 1 | TokenOwner$ You | TokenScript$ c_a_treasure_sac | SpellDescription$ Create a Treasure token.
```

Comment:
You can remove `TokenAmount$ 1 | TokenOwner$ You | ` as those are defaults - your call